Modifying Elements/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>DOM Manipulation</title>
+  </head>
+  <link rel="stylesheet" href="style.css" />
```

Comment:
`style.css` doesn't exist.

Reply:
Thanks, @lycoangelo, This is now updated.